urnik/templates/proste_ucilnice.html

```diff
@@ -16,8 +16,9 @@
 <div class='right-panel' id='proste-ucilnice-panel'>
   <div class="card">
     <div class="card-content">
-      <span class="card-title">Izberi teden</span>
-      Če želite, da se upoštevajo rezervacije, izberite teden. Če teden ni na izbiro, tisti teden še ni rezervacij.
+      <span class="card-title">Izberite teden</span>
+      Če želite videti terminske rezervacije, izberite teden.
+      Če ni na izbiro, rezervacij zanj še ne sprejemamo.
```

Comment:
Meni se je zdelo jasno, ampak mišljeno je kot `če tedna ni, še nihče ni naredil rezervacije`.

Reply:
Kako pa potem nekdo izbere teden, ki ga še ni in zanj naredi rezervacijo?

Reply:
Rezervacijo itak naredi tako, da pokliče v referat. Rezervacij prek spleta zaenkrat še nimamo.